docker-compose.yml

```diff
@@ -24,8 +24,12 @@ boulder:
       - 9380:9380 # mail-test-srv
       - 9381:9381 # mail-test-srv
     links:
+      - bhsm:boulder-hsm
       - bmysql:boulder-mysql
       - brabbitmq:boulder-rabbitmq
+bhsm:
+    build: test/
```

Comment:
Rather than put this with all the stuff under test/, let's put it alongside the other Dockerfile in test/boulder-tools.